### PR TITLE
libkb: show upgrade URI

### PR DESCRIPTION
**Pull request by maxtaco**
- plumb through from server
- also format openpgp s2k fixes

### CODE REVIEW

File: go/libkb/upgrade_instructions.go [hunk 1]
```diff
@@ -53,12 +53,14 @@ func darwinUpgradeInstructions() {
 	}
 
 	if IsBrewBuild {
-		printUpgradeCommand("brew update && brew upgrade " + packageName)
+		printUpgradeCommand(g, "brew update && brew upgrade "+packageName)
+	} else {
+		g.Log.Warning("  Please download a new version from " + upgradeURI)
```
Comment (oconnor663) [inline go/libkb/upgrade_instructions.go]: Will the two leading spaces here look weird? Just want to make sure it's intentional.

File: go/libkb/upgrade_instructions.go [hunk 2]
```diff
@@ -36,15 +36,15 @@ func linuxUpgradeInstructions() {
 	}
 
 	if hasPackageManager("apt-get") {
-		printUpgradeCommand("sudo apt-get update && sudo apt-get install " + packageName)
+		printUpgradeCommand(g, "sudo apt-get update && sudo apt-get install "+packageName)
```
Comment (oconnor663) [inline go/libkb/upgrade_instructions.go]: I'm kind of surprised `gofmt` doesn't have an opinion about space around `+`. Is there any particular style guide we're following on this one?

File: go/vendor/github.com/keybase/go-crypto/openpgp/packet/private_key.go [hunk 3]
```diff
@@ -88,8 +88,6 @@ func (pk *PrivateKey) parse(r io.Reader) (err error) {
 		return errors.UnsupportedError("deprecated s2k function in private key")
 	}
 
-
-
```
Comment (oconnor663) [inline go/vendor/github.com/keybase/go-crypto/openpgp/packet/private_key.go]: Any reason for these formatting changes in vendored code? Or is this a version bump?

**Comment by oconnor663** [comment]
Good to go with minor questions.

File: go/libkb/upgrade_instructions.go (hunk 1 above)
Comment (maxtaco) [inline go/libkb/upgrade_instructions.go]: it's similar to what `printUpgradeCommand` is doing.  Yes intentional.

File: go/libkb/upgrade_instructions.go (hunk 2 above)
Comment (maxtaco) [inline go/libkb/upgrade_instructions.go]: gotm

File: go/vendor/github.com/keybase/go-crypto/openpgp/packet/private_key.go (hunk 3 above)
Comment (maxtaco) [inline go/vendor/github.com/keybase/go-crypto/openpgp/packet/private_key.go]: these changes were made earlier and i forgot to gofmt them.  i thought we'd get it done finally

**Comment by maxtaco** [comment]
Thanks for the review!

**Comment by zanderz** [comment]
lgtm

**Pull request merged**